File: Arrays/CheckSorted.java
package Arrays;
import java.util.Scanner;
import java.util.ArrayList;
public class CheckSorted {
    static boolean CheckSorted(ArrayList<Integer>arr){
        for(int i=1;i<arr.size();i++){
            if(arr.get(i-1)>arr.get(i))
                return false;
        }
        return true;
    }
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int n = sc.nextInt();
        ArrayList<Integer> arr = new ArrayList<Integer>();
        for(int i=0;i<n;i++)
            arr.add(sc.nextInt());
        System.out.println("The array is sorted : "+CheckSorted(arr));
    }
}

File: Arrays/CheckSorted.cpp
#include<bits/stdc++.h>
using namespace std;
bool CheckSorted(vector<int>&arr){
        for(int i=1;i<arr.size();i++){
            if(arr[i-1]>arr[i])
                return false;
        }
        return true;
    }
    int main() {
        int n,a;
        cin>>n;
        vector<int>arr(n);
        for(int i=0;i<n;i++){
            cin>>a;
            arr.push_back(a);
        }
        cout<<"The array is sorted : "<<CheckSorted(arr);
    }

